packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+.rt-DataListLabel {
+  color: var(--gray-a11);
+  box-sizing: border-box;
+  display: block;
+  white-space: nowrap;
+}
+
+.rt-DataListData {
+  box-sizing: border-box;
+  margin: 0;
+  display: block;
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * ALIGN                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+@breakpoints {
+  .rt-DataListItem {
+    &:where(.rt-r-dl-vaf-start) {
+      align-items: start;
+    }
+    &:where(.rt-r-dl-vaf-center) {
+      align-items: center;
+    }
+    &:where(.rt-r-dl-vaf-end) {
+      align-items: end;
+    }
+    &:where(.rt-r-dl-vaf-baseline) {
+      align-items: baseline;
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * ORIENTATION                                                                                     *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+@breakpoints {
+  .rt-DataListRoot {
+    &:where(.rt-r-dl-orient-vertical) :where(.rt-DataListItem) {
+      display: flex;
+      flex-direction: column;
+      gap: var(--space-1);
+    }
+    &:where(.rt-r-dl-orient-horizontal) :where(.rt-DataListItem) {
+      display: grid;
+      /* Use subgrid so all the label columns remain aligned */
+      grid-template-columns: subgrid;
+      padding: 0;
+      gap: inherit;
+    }
+    &:where(.rt-r-dl-orient-vertical) :where(.rt-DataListLabel) {
+      white-space: normal;
+      padding: 0;
+      width: auto;
+    }
+  }
+}
```

Comment:
- Let's add all orientation-specific CSS in this block above `@breakpoints`, like that `grid-template-columns: auto 1fr`, so that all related CSS is compartmentalised
- Nit: similarly, I think it will be easier to read if this block is grouped by orientation rather than by part
- Nit: we'd usually do something generic like `.rt-r-orientation-horizontal` instead of classes scoped to components (`.rt-r-dl-...`)
- Could you add a fallback for when subgrid is not supported? (I assume `grid-template-columns: inherit` is what we want)
- Why do we need to set padding to 0?
- Why is `width: auto` needed?